src/renderer/components/+entity-settings/entity-settings.tsx

```diff
@@ -57,28 +58,40 @@ export class EntitySettings extends React.Component<Props> {
   async componentDidMount() {
     const { hash } = navigation.location;
 
-    this.ensureActiveTab();
+    if (hash) {
+      const item = this.menuItems.find((item) => item.title === hash.slice(1));
```

Comment:
I don't think we should be calling `.slice(1)` for every item in `this.menuItems`.

I also don't think that we should do this if `hash === "#"` which is very possible.